packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.scss

```diff
@@ -0,0 +1,4 @@
+.ajust-flavors {
```

Comment:
**thought:** We tend to avoid such custom style.

We should find a better way to handle such a case. We should report it for next iteration.